modules/benchmarks/src/synthetic.rs

```diff
@@ -0,0 +1,448 @@
+//! STDB module used for benchmarks.
```

Comment:
if you're renaming this file you also need to adjust the config used in crates/bench/src/spacetime_module

Reply:
also you'll need to rebase, everything in here has been renamed